<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windowLinkedList</class>
 <widget class="QDialog" name="windowLinkedList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Digite o número:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="entrada">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>140</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Posição:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>170</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeAfter">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>202</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove after</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="positionRemove">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="retTamanho">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tamanho da lista:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InsertFront">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>200</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert front</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeN">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="positionInsert">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="insertAfter">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>230</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert after</string>
   </property>
  </widget>
  <widget class="QPushButton" name="insertN">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert </string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeFront">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove front</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Posição:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="elem1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="elem2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="elem3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="elem4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="elem5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="elem6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="elem7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="elem8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="elem9">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="elem10">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
